src/cloud/view/cloudsmodel.cpp

```diff
@@ -143,7 +145,23 @@ QVariant CloudsModel::cloudInfo(const QString& cloudCode) const
     QString cloudTitle = m_clouds[index]->cloudInfo().title;
     AccountInfo accountInfo = m_clouds[index]->accountInfo().val;
 
-    return makeCloudInfoMap(cloudTitle, accountInfo);
+    QVariantMap cloudInfoMap = makeCloudInfoMap(cloudTitle, accountInfo);
+
+    const ui::ThemeInfo& currentTheme = uiConfig()->currentTheme();
+
+    if (cloudCode == cloud::MUSESCORE_COM_CLOUD_CODE) {
+        cloudInfoMap[prv::CLOUD_LOGO_URL] = "qrc:/resources/MuseScoreLogo.svg";
```

Comment:
I think we should store this info in m_clouds[index]->cloudInfo()
Maybe even store the URL of the logo instead of adding the file to the project (we need to think what's better)